### ISSUE TITLE
Destination API documentation and constructor signature does not match

### ISSUE THREAD

**Issue by fappel** [eclipsesource/restfuse#65]
The snippet

public Destination destination = new Destination( "http://localhost" );

described by the documentation does not compile, as this constructor signature does not exist. The additional testObject parameter is undocumented (guess this should be the surrounding test case?).

Side note: Since this is an API breakage compared to 1.1 this should be actually imply an update of the major version number ;-)

**Comment by DR-BoneZ** [comment]
yeah this threw me for a loop for a while. Took me quite a bit to realize that I should just put "this" in the testObject parameter.
